### ISSUE TITLE
Multiple cursors doesnt work correct with Vim mode plugin

### ISSUE THREAD

**Issue by yatanasovtb** [Microsoft/vscode#66335]
Steps to Reproduce:

1. Wtite similar code:
```
function ffff() {
}
$users = ffff();
$user2 = ffff();

$user3 = ffff();

```
2. While in VIM mode using VIM plugin place the cursor somewhere in the first 'ffff' word

3. Exit vim mode with 'a' or 'i' key.

4. Click 'ctrl-d' a couple of times

Expected result:
4 cursors should be created.

Actual result:
Only 2 multiple cursors are created for the first 2 occurences of the word 'ffff'

**Useful information:**
I'm not sure if this is a bug with VSCode or VIM plugin but the solution that works is this:

1. When I enter insert mode and i add some character (space or other) then e delete it - after  that when i select ctrl-d the cursors works great as expected.

Basically the cursors start working good when I edit some text anywhere in the editor and then click 'ctrl-d'.

If I go to NORMAL mode and immediately try to set multiple cursors it doesnt work correct.

**Comment by yatanasovtb** [comment]
I see there is no activity. (and maybe there will never be).

So I'm closing the issue.
